pkg/minikube/bootstrapper/kubeadm/versions.go

```diff
@@ -205,8 +205,17 @@ var versionSpecificOpts = []VersionedExtraOption{
 			Key:       "admission-control",
 			Value:     strings.Join(util.DefaultAdmissionControllers, ","),
 		},
+		LessThanOrEqual:    semver.MustParse("1.10.10"),
```

Comment:
Won't this break when there's a `1.10.11` ? Does `semver.MustParse("1.10.*")` not work?

Reply:
The semver library doesn't support *'s :(

```shell
panic: semver: Parse(1.10.*): Invalid character(s) found in patch number "*"
```

I'll change to something like 1000